### ISSUE TITLE
@405 false positive

### ISSUE THREAD

**Issue by PaymonK** [Geeksltd/GCop#92]
```c#
                var lists = GetLists(parentType, autoCreate: false);

                if (lists != null)
                    lock (lists) lists.Clear();
```

**Comment by Foroughi** [comment]
@PaymonK Would you please why this is a false alarm !? is it because inside `IF` we have something like: 

`lock ....`

**Comment by PaymonK** [comment]
Yes.
